src/runtime/windows_get_symbol.cpp

```diff
@@ -20,7 +23,12 @@ WEAK void *halide_get_symbol_impl(const char *name) {
 }
 
 WEAK void *halide_load_library_impl(const char *name) {
-    return LoadLibraryA(name);
+    // Suppress dialog windows during library open.
+    unsigned old_mode = SetErrorMode(0);
```

Comment:
Is this ever called in multi-threaded contexts? It doesn't seem thread-safe, as if errors are already suppressed, it leaves open a time in which errors could open a popup. Might be safer to do:

    unsigned old_mode = SetErrorMode(SEM_FAILCRITICALERRORS | SEM_NOOPENFILEERRORBOX);
    void *lib = LoadLibraryA(name);
    SetErrorMode(old_mode);
    return lib;

or grab a ScopedSpinLock on a global weak int.